sync-pingdom-ec2-security-groups.py

```diff
@@ -130,12 +132,16 @@ def main():
         default=80,
         help='The highest port on which Pingdom probes')
     parser.add_argument(
+        '--profile',
+        default='default',
+        help='AWS credintials profile from ~/.aws/credentials, default: default')
```

Comment:
(Typo ;).) Selecting a custom profile [also](https://boto3.readthedocs.io/en/latest/guide/configuration.html) allows one to use custom configuration settings from the `~/.aws/config` file. I suggest we phrase this more generally.

Also, `README.md` should be uodated to document this new flag.